zap/src/zap.py

```diff
@@ -307,13 +307,13 @@ def zap_compliance_scan(
     # AUTH - authenticated with SA, active scan is performed.
     # LEOAPP - authenticated with SA and registered cookie, active scan and ajax spider is performed
     # IAPUI - authenticated with iap bearer token and cookie.
-    #IAPAPI - authenticated with iap bearer token, api spec.
+    #IAPAUTH - authenticated with iap bearer token, api spec.
 
     # Set up context for scan
     context_id = zap_setup_context(zap, project, host)
 
     if scan_type != ScanType.BASELINE:
-        if scan_type == ScanType.IAPUI or scan_type == ScanType.IAPAPI:
+        if scan_type == ScanType.IAPUI or scan_type == ScanType.IAPAUTH:
```

Comment:
Would be nice to parameterize the cookie_name or maybe rename the ScanType to BEEHIVE.

Reply:
I'm not sure how I would parameterize it best, so I changed the scan type.